apps/docs/pages/guides/functions/unit-test.mdx

```diff
@@ -0,0 +1,132 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'unit-test',
+  title: 'Unit Tests for Edge Functions',
+  description: 'Unit Tests for Edge Functions with Deno Test',
+}
+To ensure the correctness and performance of your Edge Functions, testing is an essential step in the development process. Let's discuss the significance of testing and examining the code in the `deno-test.ts` file, which demonstrates how to test the Supabase client and the `hello-world` function. When creating test files for Edge Functions, it is recommended to place them in the `supabase/tests` directory and name the file by the function name followed by `-test.ts`. For example, `hello-world-test.ts`.
```

Comment:
```suggestion
}

To ensure the correctness and performance of your Edge Functions, testing is an essential step in the development process. Let's discuss the significance of testing and examining the code in the `deno-test.ts` file, which demonstrates how to test the Supabase client and the `hello-world` function. When creating test files for Edge Functions, it is recommended to place them in the `supabase/tests` directory and name the file by the function name followed by `-test.ts`. For example, `hello-world-test.ts`.
```

Reply:
This uses a test folder inside /functions/ subdirectory to create the test functions. So, I shouldn't interfere with: https://supabase.com/docs/guides/database/testing